pkgs/development/coq-modules/ssprove/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, mkCoqDerivation, coq, version ? null
+, equations
+, mathcomp
+, mathcomp-analysis
+, extructures
+, deriving
+}:
+
+(mkCoqDerivation {
+  pname = "ssprove";
+  owner = "SSProve";
+
+  inherit version;
+  defaultVersion = with lib.versions; lib.switch [coq.coq-version mathcomp.version] [
+    { cases = ["8.19" "2.2.0" ]; out = "0.2.0"; }
+    { cases = ["8.18" "2.1.0" ]; out = "0.2.0"; }
+    # This is the original dependency:
+    # { cases = ["8.17" "1.18.0"]; out = "0.1.0"; }
+    # But it is not loadable. The math-comp nixpkgs configuration
+    # will always only output version 1.18.0 for Coq 8.17.
+    # Hence, the Coq 8.17 and math-comp 1.17.0 must be explicitly set
+    # to load it.
+    # (This version is not on the math-comp CI and hence not checked.)
+    { cases = ["8.17" "1.17.0"]; out = "0.1.0"; }
+  ] null;
+
+  releaseRev = v: "v${v}";
+
+  release."0.2.0".sha256 = "sha256-GDkWH0LUsW165vAUoYC5of9ndr0MbfBtmrPhsJVXi3o=";
+  release."0.1.0".sha256 = "sha256-Yj+k+mBsudi3d6bRVlZLyM4UqQnzAX5tHvxtKoIuNTE=";
+
+  propagatedBuildInputs = [equations
+                           mathcomp
+                           mathcomp-analysis
+                           extructures
+                           deriving];
+
+  meta = with lib; {
+    description = "SSProve: A Foundational Framework for Modular Cryptographic Proofs in Coq";
+    license = licenses.mit;
+    maintainers = [ "Philipp Haselwarter <philipp@haselwarter.org>" ];
```

Comment:
I’m not sure that a string is expected there.

Reply:
Well, it expects a maintainer but I'm not sure for what.
The nixpgks-etnry or SSProve?

Reply:
This is the maintainer of the derivation in `nixpkgs` (https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md#definition-and-role-of-the-maintainer). The expected structure is documented there: https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix#L4-L15

Reply:
I'm a bit puzzled now.
I was aware of this list but do not know how to proceed here. I can take care of the maintenance of the SSProve derivation. What would be the proper steps now? Should I insert myself into the maintainers list?

It would also be ok for me if somebody of you wants to take care of this derivation.

Reply:
You can just remove the `maintainers` field. Or put yourself in:

~~~
maintainers = [ { name = "Sebastian"; github = "sertel"; } ];
~~~